IsTheMartin/mrtnmrls
# Bug fix in favicon

## Summary
`svg` format in images does not display them with
colors, only monochromatic.

`favicon.svg` was replaced for `favicon.ico`

## Evidence

|Before|After|
|---|---|
|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/a12130f8-87aa-468b-b4a3-fc6370091e4c)|![image](https://github.com/IsTheMartin/mrtnmrls/assets/38388605/0c1a4a65-181b-472e-a076-ff01ab54c951)|

## Concern Information

To test please follow the next steps:

1. Navigate to `index`
2. Observe icon in browser tab 

## Issue ticket

- https://github.com/IsTheMartin/mrtnmrls/issues/18

## Check list

- [ x ] I did add reviewers to this PR
